CDS_Medication-Medication_Cards.csv

```diff
@@ -0,0 +1,7 @@
+UUID,Medication1,Medication1 SNOMED Code,Medication2,Medication2 SNOMED Code,Card Indicator,Card Summary,Card Detail,Source,Source Link

```

Comment:
Please replace this file with tab separated values. Comma separation does not allow for ECL values.

Reply:
updated and using `tsv` file now